scopes/component/ui/version-dropdown/version-dropdown.tsx

```diff
@@ -124,13 +124,14 @@ function VersionMenu({
   }).filter((tab) => tab.payload.length > 0);
 
   const multipleTabs = tabs.length > 1;
+  const message = multipleTabs
+    ? 'Switch to view tags, snaps, or lanes'
+    : `Switch between ${tabs[0].name.toLocaleLowerCase()}s`;
```

Comment:
.toLocalLowerCase?

Reply:
is either to toLowerCase or toLocaleLowerCase (taking in consideration end user's language and region)